### ISSUE TITLE
Failed to authenticate

### ISSUE THREAD

**Issue by Borrower543** [bm1549/home-assistant-frigidaire#77]
I noticed the (official) mobile app was down yesterday, not today the app is working but I cannot authenticate via Home Assistant integration. 

`frigidaire.FrigidaireException: Failed to authenticate, sessionInfo was not in response: {'callId': 'xxxxxxx', 'errorCode': 206001, 'errorDetails': 'Registration was not finalized', 'errorMessage': 'Account Pending Registration',`

**Comment by bm1549** [comment]
@corgana if you logout of the mobile app, can you log back in?

**Comment by corgana** [comment]
Yes, today on the mobile app I can. Yesterday I logged out and was unable to log back in (couldn't get past the country selection screen).

**Comment by bm1549** [comment]
Seems like a transient frigidaire bug - going to close but re-open if it seems to be more of a problem with this integration
